ibm/conns/config.go

```diff
@@ -1169,6 +1175,11 @@ func (session clientSession) MetricsRouterV3() (*metricsrouterv3.MetricsRouterV3
 	return session.metricsRouterClient, session.metricsRouterClientErr
 }
 
+// Logs Router API
+func (session clientSession) IbmLogsRouterOpenApi30V0() (*ibmlogsrouteropenapi30v0.IbmLogsRouterOpenApi30V0, error) {
+	return session.ibmLogsRouterOpenApi30Client, session.ibmLogsRouterOpenApi30ClientErr
+}
+
```

Comment:
Add emptycredentialsError check for logs router here - https://github.com/IBM-Cloud/terraform-provider-ibm/blob/a81d5798b7a5a27b50d740633da3ec1b9851ed93/ibm/conns/config.go#L1310

Reply:
Added emptycredentialsError check for a Logs Router Client below: https://github.com/IBM-Cloud/terraform-provider-ibm/pull/4945/files#diff-971fca808334dec2e3c5f2462c1d78588f11e0feb1e4fc418dc652fda857511fR1310